source/weeknotes/2019-01-25-week-25.html.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Week 25: We've got our first customer!"
+author: Ian Drysdale
+---
+**Protecting liberty by simplifying security**
+
+_Recap_: We’re building Fluidkeys to help teams safeguard their source code and protect sensitive data.
+
+## The short version
+
+Hello, it's great to be back!
+
+* We've made [a landing page explaining how Fluidkeys works for teams](/fluidkeys-v1-preview)
+* We made our first sale! 🍾
+* We've re-jigged the road map, prioritising team management next 🛣️
+
+## Started work on Fluidkeys for Teams proposition
+
+We've had a stab at explaining how Fluidkeys will actually work for your teams. We're really narrowly targetting our niche: small to medium engineering teams. We're about to show how it looks (as a command line app) and how it works. I'm really pleased with what we've got done, [take a look](/fluidkeys-v1-preview).
+
+![A screenshot of the header of the landing page](images/2019-01-25-landing-page-header.png)
+
+We'll be testing this out with prospective users next week, seeing whether it's getting closer to being something valuable enough to pay for!
+
+Speaking of which...
+
+## We made our first sale!
+
+This week we got our first paying subscriber. Well, in principle. Now the small matter of delivering the product, figuring out how to take payment, and incorporate. Lean! It's great to have this, less so for the finances at this stage, but more as validation that we're moving in the right direction.
+
+Who knows, at somepoint the [dashboard](http://fluidkeys-dashboard.herokuapp.com/) might stop shouting about how unsustainable our business is! 🔮
+
+## Next up: team management 👯‍♂️
+
+In light of the sale and our narrower focus on teams, we've realised to finish the product we need someway for admins to define who's in their team. We've prompted this to be the next major release on our [roadmap](/#roadmap). We reckon this will be a simple text file listing team-mates and their fingerprints, signed by the team admin. On Thursday we worked up an idea that an administrator could also connect Github or Bitbucket, and pull a list of their team from there (we could also check if they've already got PGP keys). We then started sketching a flow of how the whole invite flow would work, along with the authentication.
```

Comment:
I don't think `#roadmap` is a valid anchor, there's a `#current_release` which is nearby...

Reply:
I've added it to our index page with this commit, haven't I?

Reply:
OOPS